### ISSUE TITLE
Implement TimePicker.ClockIdentifier

### ISSUE THREAD

**Issue by hartez** [dotnet/maui#441]
Determines whether the TimePicker uses a 12 or 24 hour format.

New property for ITimePicker
`string ClockIdentifier {get;}`
See also: https://docs.microsoft.com/en-us/uwp/api/windows.globalization.clockidentifiers?view=winrt-19041

Default value is determined by DateTimeFormat for the app culture. We'll also need a setter from Controls.TimePicker.

**Comment by jfversluis** [comment]
I don't think this is supported on other platforms than Windows without hacking it in there. Also don't see a lot of traction for this so closing it for now. Thanks for the suggestion!
